ui/src/components/credentials/CredentialIssuedQR.tsx

```diff
@@ -46,27 +48,30 @@ export function CredentialIssuedQR() {
 
   const onStartAgain = () => {
     makeRequestAbortable(createCredentialQR);
-    setIssuedQRCode({ status: "pending" });
+    setIssuedQRCodes({ status: "pending" });
   };
 
-  if (hasAsyncTaskFailed(issuedQRCode)) {
+  if (hasAsyncTaskFailed(issuedQRCodes)) {
     return (
       <ErrorResult
-        error={issuedQRCode.error.message}
+        error={issuedQRCodes.error.message}
         labelRetry="Start again"
         onRetry={onStartAgain}
       />
     );
   }
 
-  if (!isAsyncTaskDataAvailable(issuedQRCode)) {
+  if (!isAsyncTaskDataAvailable(issuedQRCodes)) {
     return <LoadingResult />;
   }
 
+  const [issuedQRCodeLink, issuedQRCodeRaw] = issuedQRCodes.data;
   return (
     <CredentialQR
-      qrCode={issuedQRCode.data.qrCodeLink}
-      schemaType={issuedQRCode.data.schemaType}
+      qrCodeBase64={`iden3comm://?i_m=${btoa(issuedQRCodeRaw.qrCode)}`}
```

Comment:
I'd remove this prop and move this logic to the component `CredentialQR`

Reply:
I agree. Done,